### PR TITLE
Trove.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Trove.js [hunk 1]
```diff
@@ -113,63 +116,62 @@ function scrapeNewspaper(doc, url) {
 				// Add tags
 				var tags = ZU.xpath(doc, "//ul[contains(@class,'nlaTagContainer')]/li");
 				for (var i = 0; i < tags.length; i++) {
-					tag = ZU.xpathText(tags[i], "a[not(contains(@class,'anno-remove'))]");
+					const tag = ZU.xpathText(tags[i], "a[not(contains(@class,'anno-remove'))]");
```
Comment (zuphilip) [inline Trove.js]: let

File: Trove.js [hunk 2]
```diff
@@ -254,18 +257,18 @@ function scrapeWork(doc, url) {
 				item.abstractNote = ZU.xpathText(doc, "//meta[@property='og:description']/@content");
 				
 				// Add tags
-				tags = ZU.xpath(doc, "//div[@id='tagswork']/ul/li");
+				const tags = ZU.xpath(doc, "//div[@id='tagswork']/ul/li");
```
Comment (zuphilip) [inline Trove.js]: let

File: Trove.js [hunk 3]
```diff
@@ -254,18 +257,18 @@ function scrapeWork(doc, url) {
...
 				for (var i = 0; i < tags.length; i++) {
-					tag = ZU.xpathText(tags[i], "a");
+					const tag = ZU.xpathText(tags[i], "a");
```
Comment (zuphilip) [inline Trove.js]: let

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**